dingo/bindings/bindings.cpp

```diff
@@ -134,6 +138,72 @@ double HPolytopeCPP::apply_sampling(int walk_len,
    } else if (strcmp(method, "vaidya_walk") == 0) { // vaidya walk
       uniform_sampling<VaidyaWalk>(rand_points, HP, rng, walk_len, number_of_points,
                                    starting_point, number_of_points_to_burn);
+   } else if (strcmp(method, "shake_and_bake") == 0) {
+      using clock = std::chrono::high_resolution_clock;
```

Comment:
Counting time and computing statistics (ess, psrf) should not be part of the binding function.